diff -Nur -x '*.orig' -x '*.rej' smeserver-audittools-0.0.2/root/sbin/e-smith/audittools/pseudonyms mezzanine_patched_smeserver-audittools-0.0.2/root/sbin/e-smith/audittools/pseudonyms
--- smeserver-audittools-0.0.2/root/sbin/e-smith/audittools/pseudonyms	2006-03-30 10:50:36.000000000 +1000
+++ mezzanine_patched_smeserver-audittools-0.0.2/root/sbin/e-smith/audittools/pseudonyms	2006-03-30 14:22:02.375529441 +1000
@@ -65,5 +65,5 @@
     my ($key, $warning) = @_;
 
     warn "$key: $warning\n";
-    warn "RUN: db settype pseudonym-deleted $key\n"
+    warn "RUN: db accounts settype $key pseudonym-deleted\n"
 }
