diff -Nur -x '*.orig' -x '*.rej' smeserver-audittools-0.0.2/root/sbin/e-smith/audittools/pseudonyms mezzanine_patched_smeserver-audittools-0.0.2/root/sbin/e-smith/audittools/pseudonyms
--- smeserver-audittools-0.0.2/root/sbin/e-smith/audittools/pseudonyms	2006-03-30 15:00:27.530891489 +1000
+++ mezzanine_patched_smeserver-audittools-0.0.2/root/sbin/e-smith/audittools/pseudonyms	2006-03-30 14:59:32.624227299 +1000
@@ -65,5 +65,5 @@
     my ($key, $warning) = @_;
 
     warn "$key: $warning\n";
-    warn "RUN: db accounts settype $key pseudonym-deleted\n"
+    print "RUN: db accounts settype $key pseudonym-deleted\n";
 }
diff -Nur -x '*.orig' -x '*.rej' smeserver-audittools-0.0.2/root/sbin/e-smith/audittools/virtualdomains2pseudonyms mezzanine_patched_smeserver-audittools-0.0.2/root/sbin/e-smith/audittools/virtualdomains2pseudonyms
--- smeserver-audittools-0.0.2/root/sbin/e-smith/audittools/virtualdomains2pseudonyms	2006-03-30 10:50:36.000000000 +1000
+++ mezzanine_patched_smeserver-audittools-0.0.2/root/sbin/e-smith/audittools/virtualdomains2pseudonyms	2006-03-30 14:59:43.659351178 +1000
@@ -43,5 +43,5 @@
 	next;
     }
 
-    warn "RUN: db accounts set $pseudo pseudonym Account $account\n";
+    print "RUN: db accounts set $pseudo pseudonym Account $account\n";
 }
