diff -Nur -x '*.orig' -x '*.rej' smeserver-audittools-0.0.2/root/sbin/e-smith/audittools/virtualdomains2pseudonyms mezzanine_patched_smeserver-audittools-0.0.2/root/sbin/e-smith/audittools/virtualdomains2pseudonyms
--- smeserver-audittools-0.0.2/root/sbin/e-smith/audittools/virtualdomains2pseudonyms	2006-03-30 15:19:08.091926873 +1000
+++ mezzanine_patched_smeserver-audittools-0.0.2/root/sbin/e-smith/audittools/virtualdomains2pseudonyms	2006-03-30 15:18:42.940220307 +1000
@@ -31,17 +31,31 @@
 
 for ( <VIRTUALS> )
 {
-    next unless (/(.*@.*):(.*)/);
+    next unless (/(.*@.*):\s*(\S+)\s*/);
 
-    my ($pseudo, $account) = ($1, $2);
+    my ($pseudo, $target) = ($1, $2);
 
     my $entry = $adb->get($pseudo);
 
     if ($entry)
     {
+        my $account = $entry->prop("Account");
+
+        if (not $account)
+        {
+            warn "$pseudo Account property missing or null\n";
+            next;
+        }
+
+        if ($account ne $target)
+        {
+            warn "$pseudo=>$account, but virtualdomains=>$target \n";
+            next;
+        }
+
 	warn "$pseudo exists\n";
 	next;
     }
 
-    print "RUN: db accounts set $pseudo pseudonym Account $account\n";
+    print "RUN: db accounts set $pseudo pseudonym Account $target\n";
 }
