diff -Nur -x '*.orig' -x '*.rej' smeserver-audittools-0.0.1/root/sbin/e-smith/audit-templates mezzanine_patched_smeserver-audittools-0.0.1/root/sbin/e-smith/audit-templates
--- smeserver-audittools-0.0.1/root/sbin/e-smith/audit-templates	2006-03-23 11:09:14.000000000 +1100
+++ mezzanine_patched_smeserver-audittools-0.0.1/root/sbin/e-smith/audit-templates	2006-03-23 11:59:45.200350508 +1100
@@ -22,8 +22,27 @@
 use warnings;
 
 use RPM2;
+use File::Find;
+use File::Path;
 
 # Files not owned by RPMs
 # Files modified since install by RPM
 # Templates from "non-standard" RPMs
 # Templates overridden in templates-custom
+
+for my $dir ( qw(templates templates-user) )
+{
+    find({ wanted => \&custom_templates }, "/etc/e-smith/$dir-custom");
+}
+
+sub custom_templates
+{
+    return unless -f;
+
+    my $template = $File::Find::name;
+    $template =~ s/-custom//;
+
+    print "$File::Find::name: " .
+        ((-e $template) ? "OVERRIDE" : "ADDITION") . "\n";
+}
+
