diff -Nur -x '*.orig' -x '*.rej' smeserver-audittools-0.0.2/root/sbin/e-smith/audittools/pseudonyms mezzanine_patched_smeserver-audittools-0.0.2/root/sbin/e-smith/audittools/pseudonyms
--- smeserver-audittools-0.0.2/root/sbin/e-smith/audittools/pseudonyms	2006-03-30 15:04:19.982342462 +1000
+++ mezzanine_patched_smeserver-audittools-0.0.2/root/sbin/e-smith/audittools/pseudonyms	2006-03-30 15:03:59.901760521 +1000
@@ -31,9 +31,15 @@
     my $account = $pseudonym->prop('Account');
     my $key = $pseudonym->key;
 
+    if ($key =~ /\@$/)
+    {
+        complain($key, "missing domain");
+	next;
+    }
+
     unless ($account)
     {
-        complain($key, "no Account property");
+        complain($key, "no Account property or Account property empty");
         next;
     }
 
